easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.12.0-fosscuda-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,108 @@
+easyblock = 'PythonBundle'
+
+name = 'TensorFlow'
+version = '1.12.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+toolchainopts = {'usempi': True}
+
+builddependencies = [
+    # must stick to Bazel 0.18, cfr. https://github.com/tensorflow/tensorflow/issues/23380
+    ('Bazel', '0.18.0'),
+    ('protobuf', '3.6.1'),
+]
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('wheel', '0.31.1', versionsuffix),
+    ('h5py', '2.8.0', versionsuffix),
+    ('cuDNN', '7.1.4.18'),
+    ('NCCL', '2.3.7'),
+    ('TensorRT', '4.0.1.6', versionsuffix),
+]
+
+use_pip = True
+
+exts_list = [
+    ('protobuf-python', '3.6.1', {
+        'modulename': 'google.protobuf',
+        'source_tmpl': 'protobuf-python-%(version)s.tar.gz',
+        'source_urls': ['https://github.com/google/protobuf/releases/download/v%(version)s/'],
+        'start_dir': 'python',
+        'use_pip': False,
+        'checksums': ['40f009cb0c190816a52fc21d45c26558ee7d63c3bd511b326bd85739b2fd99a6'],
```

Comment:
`checksums` should move up, right below `source_tmpl` (which should in turn be below `source_urls`)